src/status_im/data_store/realm/schemas/account/v4/core.cljs

```diff
@@ -11,6 +11,8 @@
             [status-im.data-store.realm.schemas.account.v1.request :as request]
             [status-im.data-store.realm.schemas.account.v1.tag :as tag]
             [status-im.data-store.realm.schemas.account.v1.user-status :as user-status]
+            [status-im.data-store.realm.schemas.account.v4.group :as group]
```

Comment:
You have to create a new version of schema (`v5`) because `v4` was already defined (and merged into develop) and it will crash on application startup on devices where `v4` has been already used.